src/worker/typosquat/mod.rs

```diff
@@ -0,0 +1,9 @@
+mod cache;
+mod config;
+mod database;
+
+#[cfg(test)]
+pub(super) mod test_util;
+
+pub use cache::{Cache, Error as CacheError};
+pub use database::Crate;
```

Comment:
just a small nitpick: any reason why this module lives in the `worker` module? it happens to run in the background worker with the current implementation, but I assume nothing is stopping us from using it elsewhere, right?

for similar reasons I've lately moved the CloudFront and Fastly clients out of the `worker` module too :)

Reply:
> just a small nitpick: any reason why this module lives in the `worker` module?

No; it made more sense with the original, pre-rebase module layout, but it doesn't any more. I'll move it out like those modules.